src/app/app.module.ts

```diff
@@ -6,21 +6,34 @@ import { ToastrModule } from 'ngx-toastr';
 import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
 import { AppComponent } from './app.component';
 import { AppRoutingModule } from './app.routing';
+
 import { RouterModule } from '@angular/router';
-import { AuthLayoutComponent } from './layouts/auth-layout/auth-layout.component';
-import { CommonLayoutComponent } from './layouts/common-layout/common-layout.component';
-import { ComponentsModule } from './components/components.module';
+import { FeaturesComponent } from './modules/features/features.component';
+import { SharedModule } from './modules/shared/shared.module';
 import { AngularFontAwesomeModule } from 'angular-font-awesome';
-import { JwtInterceptor } from './helpers/jwt.interceptor';
-import { ErrorInterceptor } from './helpers/error.interceptor';
+import { JwtInterceptor } from './interceptors/jwt/jwt.interceptor';
```

Comment:
What do you think about using absolutes imports bro?

Reply:
we are supposed to use absolute paths... thanks for reminding me, thats what we are to use... let change those real quick @espoirMur